web/server/vue-cli/src/components/Statistics/Overview/Reports.vue

```diff
@@ -45,28 +60,27 @@
           </v-card-title>
           <v-row>
             <v-col
-              v-for="c in reportType.cols"
-              :key="c.label"
+              v-for="columnData in reportType.cols"
+              :key="columnData.label"
               :cols="12 / reportType.cols.length"
             >
               <router-link
                 :to="{
                   name: 'reports',
                   query: {
-                    ...$router.currentRoute.query,
                     ...{
-                      'newcheck': undefined,
-                      'compared-to-open-reports-date': undefined,
+                      'detection-status': undefined,
+                      'run': runName,
+                      'newcheck': runName,
+                      'open-reports-date': dateTimeToStr(columnData.date[0]),
+                      'compared-to-open-reports-date':
+                        dateTimeToStr(columnData.date[1]),
                     },
                     ...(reportType.id === 'new' ? {
-                      'open-reports-date': dateTimeToStr(c.date[0]),
-                      'compared-to-open-reports-date':
-                        dateTimeToStr(c.date[1]),
-                      'diff-type': 'New',
-                      'review-status': ['Confirmed bug', 'Unreviewed']
+                      'review-status': ['Confirmed bug', 'Unreviewed'],
```

Comment:
Remove these filters as the outstanding/closed status of the report is calculated by the detection and closed dates and the review-status fields.